.unreleased/RELEASE_NOTES_HEADER.md.j2

```diff
@@ -0,0 +1,91 @@
+## {{ release_current }} ({{ release_date }})
+
+This release contains performance improvements and bug fixes since 
+the {{ release_previous }} release. We recommend that you upgrade at the next
+available opportunity.
+
+In addition, it includes these noteworthy features:
+* foo
+* bar
+
+**For on-premise users and this release only**, you will need to run the following SQL script after running `ALTER EXTENSION`. More details can be found in the pull request [#6786](https://github.com/timescale/timescaledb/pull/6797).
```

Comment:
Should probably add a bit more context. This will fix hypertables with comrpression with foreign key constraints. Users not using foreign keys with hypertables may skip this step.